zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/ClusteringRule.java

```diff
@@ -344,9 +344,11 @@ private Broker createBroker(final int nodeId) {
             new BrokerClusterConfiguration().clusterConfig(brokerSpringConfig),
             Version.from(VersionUtil.getVersion()));
     final var scheduler =
-        new io.camunda.zeebe.broker.ActorSchedulerConfiguration(
-                brokerSpringConfig, actorClockConfiguration)
-            .scheduler(IdleStrategySupplier.ofDefault());
+        new io.camunda.commons.actor.ActorSchedulerConfiguration(
```

Comment:
🔧 I guess as we delete the other class we don't need to reference it via package name

Reply:
I think, it gets removed in one of the next PRs.